### PR TITLE
Switched all the remaining instances of govuk blue to the new blue

### PR DESCRIPTION
**Pull request by smcmurtry**
# Summary | Résumé

I noticed that the blue being used on the dashboard was the old govuk blue, instead of the new one in figma. So I switched that, as well as all the other instances of govuk blue I could find. I ran `npm run tailwind` to compile the css files.

Note: tailwind classes with `xyz-blue` use our [default blue](https://github.com/cds-snc/notification-admin/blob/chore/fix-blue/tailwind.config.js#L52), while classes like `xyz-blue-govukblue` use [this one](https://github.com/cds-snc/notification-admin/blob/chore/fix-blue/tailwind.config.js#L54).

Old:
Screen Shot 2021-06-07 at 3 59 15 PM

New:
Screen Shot 2021-06-07 at 3 54 17 PM

### CODE REVIEW

**Comment by smcmurtry** [comment]
[Implement changes to templates' entry point](https://trello.com/c/kZ9SC53l/541-implement-changes-to-templates-entry-point)

**Comment by anikbrazeau** [comment]
Thank you, Steve, for changing all the gov-uk blues!! 🤩🤩🤩
Let me know, I can review/approve or wait for the write and send links on the dashboard 👀

**Comment by smcmurtry** [comment]
I removed the `govukblue` from `tailwind.config.py` and I also searched for the hex value for `govukblue` in the codebase and found a few more places to replace it, including the folder icon:
Screen Shot 2021-06-08 at 9 52 54 AM

File: app/config.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ class Test(Development):
 
 
 class Production(Config):
-    HEADER_COLOUR = "#005EA5"  # $govuk-blue
+    HEADER_COLOUR = "#26374A"  # blue.default
```
Comment (smcmurtry) [inline app/config.py:150]: Not used - but I replaced it anyway.

**Pull request merged**